astropy/stats/bls/__init__.py

```diff
@@ -9,8 +9,9 @@
 
 import warnings
 
-from astropy.timeseries.periodograms.bls import (BoxLeastSquares as TimeseriesBoxLeastSquares,
-                                                 BoxLeastSquaresResults as TimeseriesBoxLeastSquaresResults)
+from astropy.timeseries.periodograms.bls import BoxLeastSquares as TimeseriesBoxLeastSquares
+from astropy.timeseries.periodograms.bls import \
```

Comment:
Belated comment: Can we ask isort to not introduce `\` continuation?

Reply:
I was going to make the same comment but was too slow :)

Reply:
Ah, sorry!  Is this an `isort` configuration option?  I thought a lot of these type tools don't allow for much customization.

Reply:
Then how did this `\` happen? 🤔

Reply:
I meant an `isort` option to override this default behavior.  A lot of these tools are rigid by design (e.g., `black`).

Reply:
There is an `isort` parameter to use parenthesis instead of a line-continuation character: ``--up`` or ``--use-parenthesis``.  I agree that we should use that.

Reply:
Yes. I can reconfigure `isort` and run on it the whole repo.

Reply:
I opened #13714